### PR TITLE
test mode of batch normalization

### PR DESCRIPTION
**Pull request by varunjain99**
Translates BatchNormalization for test mode
@bddppq @yinghai @Maratyszcza

### CODE REVIEW

File: onnx_xla/translations/translate_batch_normalization.cc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+onnxStatus translateBatchNormalization(const Node& n,
+                                       XlaBuilder& builder,
+                                       ValueOpMap& valueToOp) {
+  if (n.outputs().size() > 1 || n.hasAttribute(kmomentum) ||
+      n.hasAttribute(Symbol("spatial"))) {  // TODO: ENFORCE
+    throw std::runtime_error("Only test mode of BatchNormalization supported");
+  }
+
+  float epsilon = 1e-5;
+  if (n.hasAttribute(kepsilon)) {
```
Comment (yinghai) [inline onnx_xla/translations/translate_batch_normalization.cc]: where is `kepsilon` defined?

Comment (varunjain99) [reply onnx_xla/translations/translate_batch_normalization.cc]: kepsilon is one of the predefined ONNX symbols (found here https://github.com/onnx/onnx/blob/master/onnx/common/interned_strings.h). It's included from operator_registry.h through onnx/common/ir.h

File: onnx_xla/translations/translate_batch_normalization.cc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    epsilon = n.f(kepsilon);
+  }
+  valueToOp[n.outputs().at(0)] = builder.BatchNormInference(
+      valueToOp.at(n.inputs().at(0)), valueToOp.at(n.inputs().at(1)),
+      valueToOp.at(n.inputs().at(2)), valueToOp.at(n.inputs().at(3)),
+      valueToOp.at(n.inputs().at(4)), epsilon, 1);
```
Comment (bddppq) [inline onnx_xla/translations/translate_batch_normalization.cc]: maybe add some doc here to explain what is `1` here?

File: onnx_xla/translations/translate_batch_normalization.cc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// Tranlates batch normalization in test mode
+onnxStatus translateBatchNormalization(const Node& n,
+                                       XlaBuilder& builder,
+                                       ValueOpMap& valueToOp) {
+  if (n.outputs().size() > 1 || n.hasAttribute(kmomentum) ||
+      n.hasAttribute(Symbol("spatial"))) {  // TODO: ENFORCE
+    throw std::runtime_error("Only test mode of BatchNormalization supported");
+  }
+
+  float epsilon = 1e-5;
```
Comment (bddppq) [inline onnx_xla/translations/translate_batch_normalization.cc]: add a TODO to fetch this default value from the onnx schema
